ui/admin/app/components/form/host/azure/index.hbs

```diff
@@ -52,25 +52,17 @@
     {{/if}}
   </form.textarea>
 
-  <form.fieldset>
-    <:title>{{t 'titles.provider'}}</:title>
-    <:body>
-      <form.radioGroup
-        @name='provider'
-        @selectedValue={{@model.compositeType}}
-        @disabled={{true}}
-        as |radioGroup|
-      >
-        <radioGroup.card
-          @label={{t 'resources.host-catalog.types.azure'}}
-          @value={{@model.compositeType}}
-          @layout='tile'
-          @icon={{concat 'flight-icons/svg/' @model.compositeType '-color-16'}}
-        />
-      </form.radioGroup>
-    </:body>
-  </form.fieldset>
-
+  <InfoField
+    @value={{t 'descriptions.provider'}}
+    @icon='azure-color'
+    disabled={{form.disabled}}
```

Comment:
So this references the `@disabled.isSaving` withint the line 4, is that right?

Reply:
Yeah, we will be using `form.disabled`. This is part of the [initial HDS migration work](https://github.com/hashicorp/boundary-ui/commit/02b6afa5e752d5e077c65574bace96a4a3e0c3dd).  I didn't refactor the existing fields in these forms to keep the PR simple and reviewable(as there a quite a few files and fields in this PR). I can create a tech debt ticket to track this work.  But keep in mind, this will eventually be updated when we migrate the components one by one to HDS.

Reply:
ty for asking this and adding the [ticket](https://hashicorp.atlassian.net/browse/ICU-7233)